File: Bindings~/java/src/main/java/ro/altom/altunitytester/Commands/ObjectCommand/AltTapElement.java
package ro.altom.altunitytester.Commands.ObjectCommand;

import ro.altom.altunitytester.IMessageHandler;
import ro.altom.altunitytester.AltUnityObject;
import ro.altom.altunitytester.Commands.AltBaseCommand;

public class AltTapElement extends AltBaseCommand {
    /**
     * @param command The parameters
     */
    private AltTapClickElementParams parameters;

    /**
     * @param messageHandler - Message
     * @param parameters - int count , float interval , boolean wait
     */

    public AltTapElement(IMessageHandler messageHandler,
            AltTapClickElementParams parameters) {
        super(messageHandler);
        this.parameters = parameters;
        this.parameters.setCommandName("tapElement");
    }

    public AltUnityObject Execute() {
        SendCommand(parameters);
        AltUnityObject obj = recvall(parameters, AltUnityObject.class);
        obj.setMesssageHandler(messageHandler);

        if (parameters.getWait()) {
            String data = recvall(parameters, String.class);
            validateResponse("Finished", data);
        }

        return obj;
    }
}

File: Assets/AltTester/AltDriver/Commands/ObjectCommands/AltTapElement.cs
namespace Altom.AltDriver.Commands
{
    public class AltTapElement : AltCommandReturningAltElement
    {
        AltTapElementParams cmdParams;

        public AltTapElement(IDriverCommunication commHandler, AltObject altUnityObject, int count, float interval, bool wait) : base(commHandler)
        {
            cmdParams = new AltTapElementParams(
            altUnityObject,
             count,
             interval,
             wait);
        }
        public AltObject Execute()
        {
            CommHandler.Send(cmdParams);
            var element = ReceiveAltObject(cmdParams);

            if (cmdParams.wait)
            {
                var data = CommHandler.Recvall<string>(cmdParams);
                ValidateResponse("Finished", data);
            }
            return element;
        }
    }
}